plugins/embark-assistant/embark-assistant.cpp

```diff
@@ -253,6 +253,13 @@ DFhackCExport command_result plugin_onstatechange(color_ostream &out, state_chan
 
 command_result embark_assistant(color_ostream &out, std::vector <std::string> & parameters)
 {
+    bool fileresult = false;
+
+    if (parameters.size() == 1 &&
+        parameters[0] == "fileresult") {
+        remove(fileresult_file_name);
+        fileresult = true;
+    } else
```

Comment:
Style: `else if` should be on the same line
I'll fix this, but it's easy to misinterpret the next line as a standalone `if`